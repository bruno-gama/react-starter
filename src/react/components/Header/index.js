import React from 'react'
import styled from 'styled-components'
import { Heading, Box } from 'rebass'

const Header = () => (
  <Box px={4} py={5} color="white" bg="teal">
    <Heading is="h1" fontSize={[4, 5, 6]}>
      You are beautiful
    </Heading>
    <Heading is="h2" fontSize={[3, 4, 5]}>
      Have a nice day
    </Heading>
  </Box>
)

export default Header
