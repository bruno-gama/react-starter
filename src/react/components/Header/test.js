import React from 'react'
import { render } from '@testing-library/react'

import Header from './'

describe.only('Header', () => {
  test('renders correctly', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
