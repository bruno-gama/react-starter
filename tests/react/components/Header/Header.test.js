import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Header from '../../../../src/react/components/Header'

describe.only('Header', () => {
  test('renders correctly', () => {
    const component = shallow(<Header />)

    expect(toJson(component)).toMatchSnapshot()
  })
})
