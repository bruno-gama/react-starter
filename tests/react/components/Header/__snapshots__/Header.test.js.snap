// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<Box
  bg="teal"
  color="white"
  px={4}
  py={5}
>
  <Heading
    fontSize={
      Array [
        4,
        5,
        6,
      ]
    }
    fontWeight="bold"
    is="h1"
    lineHeight={1.25}
    m={0}
  >
    You are beautiful
  </Heading>
  <Heading
    fontSize={
      Array [
        3,
        4,
        5,
      ]
    }
    fontWeight="bold"
    is="h2"
    lineHeight={1.25}
    m={0}
  >
    Have a nice day
  </Heading>
</Box>
`;
